src/app/(navigation)/home/_component/RegionSelect/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import Icon from "@/app/_component/common/Icon";
+import useModal from "@/app/hooks/useModal";
+import { cn } from "@/utils/cn";
+import { MouseEvent } from "react";
+
+const RegionSelect = () => {
+  const { Modal, open, close, isOpen } = useModal({
+    elementId: "modal-select",
+    modalType: "dropBox",
+    animate: "grow"
+  });
+
+  const handleClickStore = (e: MouseEvent<HTMLUListElement>) => {
+    if (!e.target || !(e.target instanceof HTMLLIElement)) return;
+    const targetValue = e.target.id;
+    localStorage.setItem("regionSelect", JSON.stringify(targetValue));
```

Comment:
targetValue는 문자열이기 때문에 JSON.stringify를 사용할 필요가 없을거같네요!
```suggestion
localStorage.setItem("regionSelect", targetValue);
```